<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventana_subir_PNG</class>
 <widget class="QMainWindow" name="ventana_subir_PNG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>869</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> background-color: rgb(249, 247, 245)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>221</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="subtitulo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Cargar imagen para visualizar:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_subir_img">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>431</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 10pt &quot;Calibri&quot;;  background-color: rgb(9, 139, 148)</string>
    </property>
    <property name="text">
     <string>Subir imagen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guardar">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>750</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Calibri&quot;; background-color: rgb(159, 183, 190)</string>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="atras">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>750</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Calibri&quot;; background-color: rgb(159, 183, 190)</string>
    </property>
    <property name="text">
     <string>Atras</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>509</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="logo_pricipal.qrc"/>
 </resources>
 <connections/>
</ui>
